resources/user-agents/browsers/chrome/headless-chrome.json

```diff
@@ -21,6 +21,26 @@
           "platforms": [
             "Linux_64", "Linux"
           ]
+        },
+        {
+          "match": "Mozilla/5.0 (#PLATFORM#) applewebkit* (*khtml*like*gecko*) HeadlessChrome* Safari/*",
```

Comment:
Windows support for Headless Chrome was added in version 60

Mac was added with Linux in version 59

https://developers.google.com/web/updates/2017/04/headless-chrome